app/uploaders/base_uploader.rb

```diff
@@ -13,6 +13,10 @@ def extension_whitelist
     %w[jpg jpeg jpe gif png ico bmp dng]
   end
 
+  def size_range
+    1..10.megabytes
```

Comment:
There will be an extra layer of optimization before it gets served, so I think it can be bigger.

How about 25 to start, and we can lower it if we have problems with big images?

```suggestion
    1..25.megabytes
```

Reply:
Ok thats the first time I have ever used a commit suggestion before. That was SLICK!!!